src/components/Reusables/PageContainer/PageContainer.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button, ButtonColor, ButtonSize, ButtonVariant } from '@altinn/altinn-design-system';
+import { useNavigate } from 'react-router-dom';
+
+import { ReactComponent as ExitIcon } from '@/assets/Error.svg';
+
+import classes from './PageContainer.module.css';
+
+export interface PageContainerProps {
+  children: React.ReactNode;
+}
+
+export const PageContainer = ({ children }: PageContainerProps) => {
+  const navigate = useNavigate();
+
+  return (
+    <div className={classes.pageWrapper}>
+      <div className={classes.exitButton}>
+        <Button
+          variant={ButtonVariant.Quiet}
+          color={ButtonColor.Inverted}
+          size={ButtonSize.Medium}
+          icon={<ExitIcon />}
+          onClick={() => navigate('/Profile')}
+        ></Button>
```

Comment:
Is this the right icon?
![image](https://user-images.githubusercontent.com/22012997/210835528-24439fc0-31da-45a1-8060-27759f00064d.png)
![image](https://user-images.githubusercontent.com/22012997/210835570-63496e0a-1c0e-4f41-b546-7272b768fb5d.png)